qiskit/circuit/instruction.py

```diff
@@ -653,3 +654,13 @@ def num_clbits(self):
     def num_clbits(self, num_clbits):
         """Set num_clbits."""
         self._num_clbits = num_clbits
+
+    def _compare_parameters(self, other):
+        for x, y in zip(self.params, other.params):
+            try:
+                if not math.isclose(x, y, rel_tol=0, abs_tol=1e-10):
+                    return False
+            except TypeError:
+                if x != y:
+                    return False
+        return True
```

Comment:
I guess specifically this is stdlib-gate parameters, but that's fine.

Reply:
Yeah it's why I made it private, I can add a comment or docstring for it. Or were you propoposing I rename it `_compare_stdlib_gate_parameters()`?